src/index.html

```diff
@@ -0,0 +1,93 @@
+                <div class="home">
+                    <a href="/" class="logo">Startup</a>
+                    <ul class="nav">
+                        <li><a href="#home">Home</a></li>
+                        <li><a href="#services">Services</a></li>
+                        <li><a href="#about">About</a></li>
+                        <li><a href="#works">Works</a></li>
+                        <li><a href="#blog">Blog</a></li>
+                        <li><a href="#clients">Clients</a></li>
+                        <li><a href="#contact">Contact</a></li>
+                    </ul>
+                </div>
+                <div class="head-text">
+                    <div class="head-text1">Welcome to startup</div>
+                    <div class="head-text2">You favorite creative agency template</div>
+                    <a href="/" class="start-btm">Get started</a>
+                </div>
+            </div>
+
+            <div id="services" class="section">
+                <div class="services">
+                    <div class="serv-ht">Services</div>
+                    <div class="text-services">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minim</div>
+                    <div class="serv-blocks">
+                        <div class="services-1">
+                            <div class="img-services">A</div>
+                            <div class="stn1">Clean Typography</div>
+                            <div class="st1">
+                                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minima maxime quam architecto
+                                quo inventore harum ex magni, dicta impedit. Lorem ipsum.
+                            </div>
+                        </div>
+                        <div class="services-2">
+                            <div class="img-services">Картинка №2</div>
+                            <div class="stn2">Rock Solid Code</div>
+                            <div class="st2">
+                                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minima maxime quam architecto
+                                quo inventore harum ex magni, dicta impedit. Lorem ipsum.
+                            </div>
+                        </div>
+                        <div class="services-3">
+                            <div class="img-services">Картинка №3</div>
+                            <div class="stn3">Expert Support</div>
+                            <div class="st3">
+                                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minima maxime quam architecto
+                                quo inventore harum ex magni, dicta impedit. Lorem ipsum.
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="section-color-gray">
+                <div id="about" class="section about">About us</div>
+            </div>
```

Comment:
Представь, что `section` - это основной класс, а `section-color-gray` и другие - дополнительные. В первом лежат стили общие для всех секций, а в дополнительных - более специфические. Конкретно тут не нужен сейчас внутренний `div`:

```suggestion
            <div class="section section-color-gray">About us</div>
```